dao/src/main/resources/db/changelog/logs/ch-update-into-notification-templates-Hlazova.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet id="Hlazova-6" author="Hlazova Nataliia">
+        <update tableName="notification_templates">
+            <column name="title">Bonus points added</column>
+            <where>id=32</where>
+        </update>
+        <update tableName="notification_templates">
+            <column name="title">Bonus points added</column>
+            <where>id=30</where>
```

Comment:
Why content is similar?

Reply:
<img width="587" alt="image" src="https://user-images.githubusercontent.com/87066582/179489814-5e21e9a7-a3a1-428d-986f-61550b30fad2.png">

Reply:
Is it really needed to store the duplicate data in DB?

Reply:
@VadymMakitra I'm not sure, but we have each template twice in this table.

Reply:
@NataliaHlazova But how I can see, the difference in the language id.